django_lightweight_queue/views.py

```diff
@@ -0,0 +1,78 @@
+import io
+import json
+import logging
+
+from django import template
+from django.db import transaction
+from django.conf import settings
+from django.http import Http404, HttpResponse
+
+from .job import Job
+
+
+# Note: no authentication here -- ensure you only connect up this view in
+# development!
+def debug_run(request):
+    """
+    Run a task on a `GET` request and within a transaction which will always be
+    rolled back.
+
+    This is useful for debugging tasks and for optimising their database
+    accesses (via the Django Debug Toolbar).
+
+    Logging output from the task is captured and included on the page, though
+    `print` calls are not.
+
+    You should take care when using this to debug tasks which interact with
+    services other than the database (e.g: caches or HTTP endpoints), as *only*
+    the database transaction is rolled back after the task completes. Any other
+    external requests will happen as normal.
+
+    To make this view available you'll need to have your main Django project
+    include its url somewhere. You are *strongly* encouraged only to do this in
+    DEBUG mode:
+    ```
+    if settings.DEBUG:
+        urlpatterns += (
+            url(r'', include('django_lightweight_queue.urls', namespace='django-lightweight-queue')),
+        )
+    ```
+    """
+
+    if not settings.DEBUG:
+        raise Http404("Debug view only available when DEBUG=True")
```

Comment:
Should we even divulge the existence of the view in this case? Whilst it's a 404, the message communicates that something is _supposed_ to be here.

```suggestion
        raise Http404
```
I'd leave it as this

Reply:
I don't think this text is visible to users, though I've always imagined that it gets logged somewhere. I tend to end up using the text as documentation rather than anything. Would be useful to know what Django does with this in a default install.

Reply:
Given that this urlconf isn't imported by Django by default, I'm going to leave this as-is. I think the improved clarity is worth the small risk of this message being user-visible.